### PR TITLE
Migrate module configuration in RecoJets to use default cfipython

### PR DESCRIPTION
**Pull request by jeongeun**
#### PR description: 
Optimization of the python configurations: Improve maintainability by cleaning up the duplicated and cloning from the default/reference configurations. (The previous PRs : #33207, #33307, #33352, #33543, #33563, #33671, #34007, #34091, #34009, #34155, #34371, #34480)
 
In this PR, 3 files changed.  
        modified:   RecoJets/JetProducers/python/ak4PFJetsBetaStar_cfi.py
        modified:   RecoJets/JetProducers/python/hltMVAJetPuId_cff.py
        modified:   RecoJets/JetProducers/python/hltak4CaloJets_cfi.py

Update
- Replace explicit configuration with a reference from cfipython/. (migrating EDProducer("type", .. -> typeDefaylt.clone() ) 
- Remove the type specifications already presented in cfipython/fillDescriptions reference for improved syntax safety.
- Remove the duplicated parameters that are exactly the same value in cfipython reference.

#### PR validation:
Event Content comparison check was also done and there is no change with these updates.
Tested in CMSSW_12_0_X, the basic test all passed in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html).

### CODE REVIEW

**Comment by perrotta** [comment]
please test

File: RecoJets/JetProducers/python/hltak4CaloJets_cfi.py [hunk 1]
```diff
@@ -1,61 +1,16 @@
 import FWCore.ParameterSet.Config as cms
+import RecoJets.JetProducers.FastjetJetProducer_cfi _mod
```
Comment (jpata) [inline RecoJets/JetProducers/python/hltak4CaloJets_cfi.py:2]:
```suggestion
import RecoJets.JetProducers.FastjetJetProducer_cfi as _mod
```

**Review by jpata — COMMENTED** [review]
Thanks for this! Can you double check the syntax error above? Also, could you check if the validations were affected by this error?

**Comment by perrotta** [comment]
> Thanks for this! Can you double check the syntax error above? Also, could you check if the validations were affected by this error?

Yes, the python3 error reported in the tests was due exactly to the typo you pointed out

**Comment by jpata** [comment]
> Event Content comparison check was also done and there is no change with these updates.
Tested in CMSSW_12_0_X, the basic test all passed in the CMSSW PR instructions.

right, I mean to double check if the validations done in the PR description are up to date with what was committed.

**Comment by perrotta** [comment]
please test

**Comment by jpata** [comment]
+reconstruction
- technical
- no reco differences expected/observed

**Comment by perrotta** [comment]
+1

**Pull request merged**